src/System.Management.Automation/engine/parser/token.cs

```diff
@@ -592,46 +598,51 @@ public enum TokenFlags
         /// <summary>
         /// The precedence of the logical operators '-and', '-or', and '-xor'.
         /// </summary>
-        BinaryPrecedenceLogical = 1,
+        BinaryPrecedenceLogical = 0x1,
 
         /// <summary>
         /// The precedence of the bitwise operators '-band', '-bor', and '-bxor'
         /// </summary>
-        BinaryPrecedenceBitwise = 2,
+        BinaryPrecedenceBitwise = 0x2,
 
         /// <summary>
         /// The precedence of comparison operators including: '-eq', '-ne', '-ge', '-gt', '-lt', '-le', '-like', '-notlike',
         /// '-match', '-notmatch', '-replace', '-contains', '-notcontains', '-in', '-notin', '-split', '-join', '-is', '-isnot', '-as',
         /// and all of the case sensitive variants of these operators, if they exists.
         /// </summary>
-        BinaryPrecedenceComparison = 3,
+        BinaryPrecedenceComparison = 0x5,
```

Comment:
Since enum's are treated like constants, this would be a breaking change for compiled projects right?

Reply:
Yes, this will be a breaking change. We are expecting the impact to very small though. I have update the PR description with more details.